returnn/tf/native_op.py

```diff
@@ -74,6 +75,7 @@ class OpMaker(object):
   global_lock = RLock()
   mod_cache = {}  # cache_key -> mod
   op_cache = {}  # cache_key -> op
+  log_stream = sys.stdout  # type: io.TextIOBase
```

Comment:
Better `typing.TextIO`. Also fixes the warning.
```suggestion
  log_stream = sys.stdout  # type: typing.TextIO
```
Or just remove the type at all. It is clear already just from the code.
```suggestion
  log_stream = sys.stdout
```

Reply:
This is an Issue with the type checking. All cases, (`sys.stdout`, `open`, and `StringIO`) are based on `io.TextIOBase`. For some reason, PyCharm does not complain when I use `_io.TextIOBase`, but for `io.TextIOBase`. Maybe `typing.TextIO` solves the Issue.